tscert.go

```diff
@@ -79,11 +84,12 @@ var (
 // DoLocalRequest may mutate the request to add Authorization headers.
 func DoLocalRequest(req *http.Request) (*http.Response, error) {
 	tsClientOnce.Do(func() {
-		tsClient = &http.Client{
-			Transport: &http.Transport{
+		if TailscaledTransport == nil {
+			TailscaledTransport = &http.Transport{
```

Comment:
This package global's fine and defining its nil to mean automatic is fine, but let's not write back to this global.

```go
   tr := TrailscaledTransport
   if tr == nil { 
       tr = &http.Transport{DialContext: TailscaledDialer}
   }
   tsClient = &http.Client{Transport: tr}
```

Reply:
done